org.uqbar.project.wollok.lib/src/wollok/lang.wlk

```diff
@@ -1065,89 +1138,32 @@ class Number {
 	 * -14.6165.truncate(3) ==> -14.616
 	 * 5.truncate(3) ==> 5
 	 */
-	method truncate(_decimals)
+	method truncate(_decimals) native
 
 	method plus() = self
-}
-
-/**
- * @author jfernandes
- * @since 1.3
- * @noInstantiate
- */
-class Integer inherits Number {
-	/**
-	 * The whole wollok identity implementation is based on self method
-	 */
-	override method ===(other) native
-
-	method +(other) native
-	method -(other) native
-	method *(other) native
-	method /(other) native
-	
-	/** Integer division between self and other
-	 *
-	 * Example:
-	 *		8.div(3)  ==> Answers 2
-	 * 		15.div(5) ==> Answers 3
-	 */
-	method div(other) native
 	
 	/**
-	 * raisedTo
-	 * 		3 ** 2 ==> Answers 9
-	 */
-	method **(other) native
-	
-	/**
-	 * Answers remainder of division between self and other
+	 * Former Integer methods 
 	 */
-	method %(other) native
-	
-	/** String representation of self number */
-	override method toString() native
-	
-	/** Self as a String value. Equivalent: toString() */
-	override method stringValue() native	
-	
+	 
 	/** 
 	 * Builds a Range between self and end
 	 * 
 	 * Example:
-	 * 		1..4   Answers ==> a new Range object from 1 to 4
+	 * 		1..4      Answers ==> a new Range object from 1 to 4
+	 *      1.0..4.0  Answers ==> a new Range object from 1 to 4
```

Comment:
Given that there is now no difference between 1 and 1.0, I think that this second example only adds confusion.